modules/core/client/app/init.js

```diff
@@ -12,7 +12,23 @@ angular.module(ApplicationConfiguration.applicationModuleName).config(['$locatio
   }
 ]);
 
-angular.module(ApplicationConfiguration.applicationModuleName).run(function ($rootScope, $state, Authentication) {
+angular.module(ApplicationConfiguration.applicationModuleName).run(function ($rootScope, $state, $http, Authentication, CORE_CONST) {
+
+  //set token header
+  var auth_token = localStorage.getItem('auth_token');
```

Comment:
I see conflicting information out on a lot of blogs. Maybe this is due to wanting to deploy iin different environments (web app vs. cordova/phonegap/ionic app)... 

There are some references to not caching sensitive data in Phonegap/Cordova here:
http://docs.phonegap.com/en/3.5.0/guide_appdev_security_index.md.html

It seems to me that *encrypting* the token would be the best practice if we cache it in `localStorage`
> Encrypt the refresh token and store it encrypted in the localstorage. You can use CryptoJS (a JS library to encrypt/decrypt the data) to encrypt your token using AES (see this example) and it also has an angularjs module

http://security.stackexchange.com/questions/71878/cordova-phonegap-refreshtoken-in-localstorage

Stormpath uses cookies for storing the token and their existence depends on security best practices. However, I am not sure how that blog is considering deploying a Cordova client. Again, I will ask them about this.